codechefcli/problems.py

```diff
@@ -351,30 +355,34 @@ def get_ratings(country, institution, institution_type, page, lines):
     resp = {'code': 503}
 
     if req_obj.status_code == 200:
-        ratings = req_obj.json().get('list')
-        data_rows = [RATINGS_TABLE_HEADINGS]
-
-        for user in ratings:
-            temp = []
-            temp.append(str(user['global_rank']) + "(" + str(user['country_rank']) + ")")
-            temp.append(user['username'])
-            temp.append(str(user['rating']))
-            temp.append(str(user['diff']))
-            data_rows.append(temp)
-
-        if len(ratings) == 0:
-            resp = {
-                'code': '404',
-                'data': 'Oops! we don\'t have data.',
-                'data_type': 'text'
-            }
-        else:
-            resp = {
-                'code': 200,
-                'data': data_rows,
-                'data_type': 'table'
-            }
-
+        try:
```

Comment:
Is this block fixing the issue [#21](https://github.com/sk364/codechef-cli/issues/21)?

Reply:
I think so...
The issue was that there was some problem with there sever due to many requests so json was returning an empty list. So if there is an empty list this will do the job i think.

Reply:
I think this should be solved in a separate PR. Please remove this code from this PR.